tests/test_vqe.py

```diff
@@ -184,14 +188,56 @@ def test_hamiltonian_valid_init(self, coeffs, ops):
         H = qml.vqe.Hamiltonian(coeffs, ops)
         assert H.terms == (coeffs, ops)
 
+    @pytest.mark.parametrize(
+        ("hamiltonian", "map", "new_coeffs", "new_ops", "new_wires"),
+        [
+            (
+                qml.Hamiltonian([1, 1], [qml.PauliX(0), qml.PauliZ(1)]),
+                {0: 5, 1: 3},
+                [1, 1],
+                [["PauliX"], ["PauliZ"]],
+                [[5], [3]],
+            ),
+            (
+                qml.Hamiltonian([1, 1], [qml.PauliX(0) @ qml.PauliZ(1), qml.PauliZ(1)]),
+                {0: 5, 1: 3},
+                [1, 1],
+                [["PauliX", "PauliZ"], ["PauliZ"]],
+                [[5, 3], [3]],
+            ),
+            (
+                qml.Hamiltonian(
+                    [1, 2, 1],
+                    [
+                        qml.PauliX(0) @ qml.PauliZ(1),
+                        qml.Hermitian(np.array([[1, 0], [0, 0]]), 1) @ qml.PauliZ(0),
+                        qml.PauliX(4),
+                    ],
+                ),
+                {0: 5, 1: 3},
+                [1, 2, 1],
+                [["PauliX", "PauliZ"], ["Hermitian", "PauliZ"], ["PauliX"]],
+                [[5, 3], [3, 5], [4]],
+            ),
+        ],
+    )
+    def test_hamiltonian_map_wires(self, hamiltonian, map, new_coeffs, new_ops, new_wires):
```

Comment:
What does this test check? Good practice to include a docstring explaining what it is testing/validating!